### PR TITLE
Add hello world

### PR DESCRIPTION
**Pull request by LeandroSantti**

### CODE REVIEW

File: Ejercicio 1/Ejercicio 1/Program.cs [hunk 1]
```diff
@@ -10,6 +10,8 @@ class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("Helloooo World");
+
```
Comment (demian44) [inline Ejercicio 1/Ejercicio 1/Program.cs]: Please delete this empty line.

File: Ejercicio 1/Ejercicio 1/Program.cs [hunk 2]
```diff
@@ -10,6 +10,8 @@ class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("Helloooo World");
```
Comment (demian44) [inline Ejercicio 1/Ejercicio 1/Program.cs:13]: I suggest you add a ReadKey method to stop the application and can see the message.

**Review by demian44 — CHANGES_REQUESTED** [review]

File: Ejercicio 1/Ejercicio 1/Program.cs (hunk 1 above)
Comment (LeandroSantti) [reply Ejercicio 1/Ejercicio 1/Program.cs]: Done.

File: Ejercicio 1/Ejercicio 1/Program.cs (hunk 2 above)
Comment (LeandroSantti) [reply Ejercicio 1/Ejercicio 1/Program.cs:13]: Done.

**Review by demian44 — APPROVED** [review]

**Pull request merged**